specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2022-09-01/containerInstance.json

```diff
@@ -0,0 +1,2402 @@
+            "resourceType",
+            "location"
+          ],
+          "description": "The list of capabilities."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of capabilities."
+        }
+      }
+    },
+    "Capabilities": {
+      "description": "The regional capabilities.",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource type that this capability describes."
+        },
+        "osType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The OS type that this capability describes."
+        },
+        "location": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource location."
+        },
+        "ipAddressType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The ip address type that this capability describes."
+        },
+        "gpu": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The GPU sku that this capability describes."
+        },
+        "capabilities": {
+          "type": "object",
+          "readOnly": true,
+          "description": "The supported capabilities.",
+          "properties": {
+            "maxMemoryInGB": {
+              "type": "number",
+              "readOnly": true,
+              "description": "The maximum allowed memory request in GB."
+            },
+            "maxCpu": {
+              "type": "number",
+              "readOnly": true,
+              "description": "The maximum allowed CPU request in cores."
+            },
+            "maxGpuCount": {
+              "type": "number",
+              "readOnly": true,
+              "description": "The maximum allowed GPU count."
+            }
+          }
+        }
+      }
+    },
+    "ContainerGroupSku": {
+      "description": "The container group SKU.",
+      "type": "string",
+      "enum": [
+        "Standard",
+        "Dedicated"
+      ],
+      "x-ms-enum": {
+        "name": "ContainerGroupSku",
+        "modelAsString": true
+      }
+    },
+    "EncryptionProperties": {
+      "description": "The container group encryption properties.",
+      "type": "object",
+      "properties": {
+        "vaultBaseUrl": {
+          "description": "The keyvault base url.",
+          "type": "string"
+        },
+        "keyName": {
+          "description": "The encryption key name.",
+          "type": "string"
+        },
+        "keyVersion": {
+          "description": "The encryption key version.",
+          "type": "string"
+        },
+        "identity": {
+          "type": "string",
+          "description": "The keyvault managed identity."
+        }
+      },
+      "required": [
+        "vaultBaseUrl",
+        "keyName",
+        "keyVersion"
+      ]
```

Comment:
Is this new property required for every new request? Or are any of these required properties no longer required with this change?

Reply:
Identity is an optional field, which is not always required. The vaultBaseUrl, keyName, and keyVersion are still required fields as before.